__tests__/app.test.js

```diff
@@ -19,3 +19,26 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test('200: returns an array of all topics', () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body}) => {
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((obj) => {
+            expect(obj).toMatchObject({
+                description: expect.any(String),
+                slug: expect.any(String),
+        })
+      })
+    })
+  })
+  test('404: returns a 404 error if path is incorrect', () => {
```

Comment:
Please separate this test from this describe block as it does not directly relate to this endpoint. What message are you wanting to receive from a non-existent endpoint?